src/app/modules/consultations/consultation-profile/profile-card/profile-card.component.scss

```diff
@@ -241,29 +241,28 @@ $font-family-encode-sans: 'Encode Sans', sans-serif;
         text-align: left;
         color: #151515;
         text-decoration: underline;
+        text-underline-offset: 4px;
       }
 
       .response-count {
         font-size: 13px;
-        color: #666;
+        color: #929292;
         font-weight: 700;
         line-height: 16.25px;
         font-family: $font-family-encode-sans;
         letter-spacing: 0.035em;
         text-align: right;
         margin-left: auto;
+        text-align: center;
+        margin-top: auto;
+        margin-bottom: auto;
       }
     }
 
   }
 
   .divider {
     border: 1px solid #CAE5F5;
-    margin: 0 0 16px;
-  }
-
-  .divider-one{
-    border: 1px solid #CAE5F5;
-    margin-top: 20px;
+    margin: 16px -24px;
```

Comment:
Don't use negative margins. If this change is for a single divider just create a separate class for it with a positive `margin` value

Reply:
Bro, this change is for all dividers, negative margin is given to counter the padding that is given to the container, the 
dividers should be till the border.